samples/app/MeshAssimp.h

```diff
@@ -57,6 +65,12 @@ class MeshAssimp {
         return mRenderables;
     }
 
+    //For use with normalizing coordinates
+    math::float3 minBound = math::float3(1.0f);
+    math::float3 maxBound = math::float3(-1.0f);
+
+    filament::TransformManager::Instance rootInstance = filament::TransformManager::Instance();
+    utils::Entity rootEntity;
```

Comment:
These 4 fields should perhaps be privatized and use the mFoo naming convention.

Reply:
I'm using these fields externally in gltf_viewer, so I'll keep them public.